ehitchhiking-ui/src/app/shared/components/active-trip-additional-info/active-trip-additional-info.component.html

```diff
@@ -67,7 +73,13 @@
 </div>
 <div class="for-driver" *ngIf="trip.role === userState.Driver">
 	<mat-card-title class="center title">
-		Passengers<button mat-raised-button color="accent" class="show-trip-button" (click)="showTrip()">
+		Passengers<button
+			mat-raised-button
+			color="accent"
+			matTooltip="show trip"
```

Comment:
this button has a text that absolutely equals to tooltip message. Tooltip isn't required here